### PR TITLE
Use an SVG logo across the docs

### LINKED ISSUE #5677: [TODO] Use an SVG logo in the Sphinx docs
🐣 **Is your feature request related to a problem? Please describe.**

I noticed that the sphinx docs use a non-scalable png file as a logo. It needs to be changed.

💡 **Describe the solution you'd like**

Use https://github.com/aio-libs/aiohttp/blob/master/docs/aiohttp-plain.svg in sphinx instead.

❓ **Describe alternatives you've considered**

N/A

📋 **Additional context**

N/A

### PR DESCRIPTION
**Pull request by himanshu007-creator**
## What do these changes do?

This patch switches the png logo with an SVG one.

## Are there changes in behavior for the user?

The logo in the docs looks nicer when zoomed in.

## Related issue number

Resolves #5677

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names.
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

**Comment by webknjaz** [comment]
@himanshu007-creator thanks for looking into this. While I see that this change is useful, that issue was about the sphinx docs logo set in the `docs/conf.py`. Could you please include that change here too?

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator thanks for looking into this. While I see that this change is useful, that issue was about the sphinx docs logo set in the `docs/conf.py`. Could you please include that change here too?

Will do it today👍

**Comment by webknjaz** [comment]
Looks like more changes are needed to make this work. It currently generates a link to a non-existent file (which is visible here https://aiohttp--5678.org.readthedocs.build/en/5678/). I suppose it may be necessary to add an `html_logo` var so sphinx would copy it to the static dir during the build.

**Comment by himanshu007-creator** [comment]
> Looks like more changes are needed to make this work. It currently generates a link to a non-existent file (which is visible here https://aiohttp--5678.org.readthedocs.build/en/5678/). I suppose it may be necessary to add an `html_logo` var so sphinx would copy it to the static dir during the build.

okay, so i will create a  var
`html_logo="https://raw.githubusercontent.com/aio-libs/aiohttp/4d58400477331704132cc9427d66699e5815ee25/docs/aiohttp-plain.svg"`

Then assign <code>"logo":html_logo</code>

is this the correct way?

**Comment by webknjaz** [comment]
No, only use a file name in that var, no need to have a URL coupled to GitHub. Check the Sphinx docs for more details, if necessary.

**Comment by himanshu007-creator** [comment]
> No, only use a file name in that var, no need to have a URL coupled to GitHub. Check the Sphinx docs for more details, if necessary.

can you tell the required changes please

File: README.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Async http client/server framework
 ==================================
 
-.. image:: https://raw.githubusercontent.com/aio-libs/aiohttp/master/docs/_static/aiohttp-icon-128x128.png
+.. image:: https://raw.githubusercontent.com/aio-libs/aiohttp/4d58400477331704132cc9427d66699e5815ee25/docs/aiohttp-plain.svg
```
Comment (webknjaz) [inline README.rst]: Let's keep the branch name here
```suggestion
.. image:: https://raw.githubusercontent.com/aio-libs/aiohttp/master/docs/aiohttp-plain.svg
```

File: docs/conf.py [hunk 2]
```diff
@@ -145,8 +145,9 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
+html_logo="docs/aiohttp-plain.svg"
```
Comment (webknjaz) [inline docs/conf.py]: Use a relative path in the logo definition
```suggestion
html_logo = "aiohttp-plain.svg"
```

**Review by webknjaz — APPROVED** [review]
Thanks!

**Pull request merged**